integration-tests/utils/templates/cl_node.go

```diff
@@ -0,0 +1,29 @@
+package templates
+
+import (
+	"github.com/google/uuid"
+)
+
+type NodeSecretsTemplate struct {
+	PgDbName string
+	PgHost   string
+	PgPort   string
+}
+
+func (c NodeSecretsTemplate) String() (string, error) {
```

Comment:
We can't follow the convention of marshaling TOML config from structs here because `NewSecretURL` redacts the secrets. Core doesn't use the marshaling in tests, only in CLI. No interface, it's a strict type.
Do we need a workaround or it's okay this way?

Reply:
@jmank88 please check if it's okay. We can skip our convention if it doesn't make sense and revert.
https://github.com/smartcontractkit/chainlink/pull/10067/commits/fcf3dd16522bb73efcc0d5e1bcf3b6bbfc9752cf

Reply:
Reverted, for now, too many broken tests.

Reply:
Or we can have a flag with an old type, like `TEST_NO_SECRET_MASK`.